src/runtime/libfuzzer.go

```diff
@@ -6,40 +6,86 @@
 
 package runtime
 
-import _ "unsafe" // for go:linkname
+import "unsafe" // for go:linkname
 
-func libfuzzerCall(fn *byte, arg0, arg1 uintptr)
+const retSledSize = 512
```

Comment:
Perhaps we should add something like `// Keep in sync with the definition of ret_sled in ...` here and a similar comment in the ASM file.